generate_schedule.py

```diff
@@ -199,8 +199,10 @@ def generate_schedule(args):
 
     ts_per_hour = datetime.timedelta(hours=1) / s.interval
 
+    assert len(s.constants.grid_connectors) == 1, "Only one grid connector supported"
```

Comment:
how easy is it to extend the functionality to multiple GCs? This is something for the future, not for the current PR